java/RSPEC-1862.java

```diff
@@ -0,0 +1,11 @@
+class ElifRepeatedCondition {
+    void test() {
+        int a = 1;
+
+        if (a == 1) {}
+        else if (a == 2) {}
+        else if (a == 1) {}
```

Comment:
RSPEC-1862: Else condition repeated
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._